File: 딜리버리히어로/2.py
S = "4 5 6 - 7 +"
def solution(S):
    # write your code in Python 3.6
    arr = S.split(" ")
    stack = []
    for i in arr:
        if i == "POP":
            if stack:
                stack.pop()
            else:
                return -1
            if not stack:
                return -1
        elif i == "DUP":
            stack.append(stack[-1])
        elif i == "+":
            if stack:
                a = int(stack.pop())
            else:
                return -1
            if stack:
                b = int(stack.pop())
            else:
                return -1
            if a+b <= 1048575:
                stack.append(a+b)
            else:
                return -1
        elif i == "-":
            if stack:
                a = int(stack.pop())
            else:
                return -1
            if stack:
                b = int(stack.pop())
            else:
                return -1
            if a-b < 0:
                return -1
            else:
                stack.append(a-b)

        else:
            stack.append(i)
    return stack[-1]
print(solution(S))

File: 패스트캠퍼스/네이버/2.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A) {
  // write your code in JavaScript (Node.js 8.9.4)
  let sum = 0;
  let customSum = 0;
  for (let i = 1; i < A.length + 1; i++) {
    sum += i;
  }
  for (let i = 0; i < A.length; i++) {
    customSum += A[i];
  }
  if (Math.abs(sum - customSum) > 1000000000) return -1;
  return Math.abs(sum - customSum);
}

console.log(solution([6, 2, 3, 5, 6, 10000000000]));
